src/proposal.mligo

```diff
@@ -315,7 +316,7 @@ let handle_proposal_is_over
 // Flush all proposals that passed their voting stage.
 let flush(n, config, store : nat * config * storage): return =
   if n = 0n
-  then (failwith("BAD_ENTRYPOINT_PARAMETER") : return)
+  then (failwith bad_entrypoint_parameter : return)
```

Comment:
This is now the only use left of this error, but given that we also have `empty_flush` IMO we could throw that one here as well and remove the need for this `bad_entrypoint_parameter` error.

Reply:
Removed and replace with `empty_flush`.